tests/tools/kopiarunner/kopia_snapshotter.go

```diff
@@ -28,7 +28,7 @@ const (
 	noCheckForUpdatesFlag   = "--no-check-for-updates"
 	noProgressFlag          = "--no-progress"
 	parallelFlag            = "--parallel"
-	retryCount              = 180
+	retryCount              = 300
```

Comment:
```suggestion
	retryCount              = 900
```

How about we give it a generous bump to 15 minutes or so? That gives plenty of wiggle room to recover in case we're extra slow sometimes.